### PR TITLE
adds civic.json

### PR DESCRIPTION
**Pull request by noslouch**
# What's in this PR?
Adds a civic.json file

## References
[civic.json](http://open.dc.gov/civic.json/) is an open spec for civc tech projects. The idea looks to be provide a way for civic tech aggregators to spread the word about projects of these kinds.

### CODE REVIEW

File: civic.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "politicalEntity":"",
+  "type":"Web App",
+  "needs": [
+    {"need": "Ember developer"},
+    {"need": "Phoenix developer"},
+    {"need": "Open source evangelist"}
+  ],
+  "categories": [
+    {"category": "Open Source"},
+    {"category": "Developer productivity"}
```
Comment (JoshSmith) [inline civic.json]: @noslouch maybe change to "Developer Tools"?

File: civic.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  ],
+  "tags": [
+    "open source",
+    "project management",
+    "tech for good"
```
Comment (JoshSmith) [inline civic.json]: Are these tags and the categories above already in use on their site?

Comment (noslouch) [inline civic.json]: no, they're placeholder

Comment (noslouch) [inline civic.json]: if you had any, that would be great. I'm trying to carve out some time for CC between work and life, but I can add a few thoughts if you get the ball rolling.

Comment (JoshSmith) [inline civic.json]: @noslouch not off the top of my head. Can circle back, but those look good honestly.

**Pull request merged**